### PR TITLE
add unit test for normalize module, add doc for JiebaTokenizer

### PR DESCRIPTION
**Pull request by liuxiaoan8008**
The first time to think and write python code in a standard way.  Making documentation, testing into consideration. I saw an interesting in [Sphinx](http://www.sphinx-doc.org/en/master/) website: "Cheers for a great tool that actually makes programmers want to write documentation!" haha

### CODE REVIEW

**Comment by liuxiaoan8008** [comment]
guolguol

File: jigsaw-toxic-comment-classification-challenge.py [hunk 1]
```diff
@@ -6,9 +6,8 @@
 from sklearn.feature_extraction.text import CountVectorizer
 from sklearn.pipeline import Pipeline
 from sklearn.linear_model import LogisticRegression
-from sklearn_plus.preprocessing.text.zh import Normalizer
```
Comment (ybbaigo) [inline jigsaw-toxic-comment-classification-challenge.py]: 这个文件直接删掉？当时的出发点也只是一个demo文件

File: run_demo.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    #                summary_dir="/Users/liuxiaoan/Downloads/sklearn_plus_test_cp")
+    # clf.load('/Users/liuxiaoan/Downloads/sklearn_plus_test_cp')
+    # df = pd.read_csv('/Users/liuxiaoan/Downloads/test.csv', dtype=object)
+    #
+    # print(clf.predict(df['comment_text'][:10]))
+    x_vals = np.concatenate((np.random.normal(-1, 1, 50), np.random.normal(3, 1, 50)))
+    y_vals = np.concatenate((np.repeat(-1., 50), np.repeat(1., 50)))
+    print 'data is ready...'
+
+    clf = DemoClassifier(summary_dir='/Users/liuxiaoan/Downloads/sklearn_plus_test',
```
Comment (ybbaigo) [inline run_demo.py]: 这里留相对路径？否则别人跑不起来

File: sklearn_plus/neural_network/text_classification/demo.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
```
Comment (ybbaigo) [inline sklearn_plus/neural_network/text_classification/demo.py]: 这个文件后续的定位是？

File: README.rst [hunk 4]
```diff
@@ -16,7 +16,7 @@ Sklearn Plus
```
Comment (ybbaigo) [inline README.rst]: 我看整个代码库是通过代码规范库处理过的？这方面是否有必要在readme里面明确一下具体要求？避免大家都用自己的代码规范，每次提交改动很多，但都是格式的改动。

File: jigsaw-toxic-comment-classification-challenge.py (hunk 1 above)
Comment (liuxiaoan8008) [reply jigsaw-toxic-comment-classification-challenge.py]: 可以。我明天或者后天抽空弄一下。。

File: run_demo.py (hunk 2 above)
Comment (liuxiaoan8008) [reply run_demo.py]: 好的。

File: sklearn_plus/neural_network/text_classification/demo.py (hunk 3 above)
Comment (liuxiaoan8008) [reply sklearn_plus/neural_network/text_classification/demo.py]: 这个后续会删掉。严格来讲，这一版只是验证跑通思路，并没有形成一个真正可用的模型。下周我会梳理出来至少1个可用的分类模型。

File: README.rst (hunk 4 above)
Comment (liuxiaoan8008) [reply README.rst]: 是的，这个是需要有的，这个我在gitlab那个issue里的下一步工作也有写。
